index.html

```diff
@@ -3731,6 +3731,20 @@ <h4 id="choice_of_sides_for_ruby_with_respect_to_base_characters">
         </figcaption>
 
       </figure>
+      <p its-locale-filter-list="en" lang="en">For positioning of characters of ruby annotation on block direction, 
+        character frames of ruby annotation are contiguous with ones of its base text ([[[#fig2_3_9]]]).</p>
+      <p its-locale-filter-list="ja" lang="ja">なお，ルビの行送り方向の配置位置は，親文字の文字の外枠とルビ文字の外枠を接して配置する（[[[#fig2_3_9]]]）．</p>
+      <aside class="note" id="n20211126001">
+        <p its-locale-filter-list="en" lang="en">Positioning of characters of ruby annotation on block direction</p>
+        <p its-locale-filter-list="ja" lang="ja">行送り方向のルビの配置位置</p>
+        <p its-locale-filter-list="en" lang="en">In generic principles, 
+          character frames of ruby annotation are contiguous with ones of its base text in the block direction.
+          For visually handicapped person, there are cases to desire some spacing between two character frames in the block direction, 
+          so it is desired to have capability to configure these spacing. </p>
+        <p its-locale-filter-list="ja" lang="ja">行送り方向のルビの配置位置は，親文字の文字の外枠とルビ文字の文字の外枠を接して配置するのが原則である．
+          しかし，視覚障碍者にとって，親文字とルビの行送り方向の間隔がある程度は空いているのが望ましい場合もあり，
+          この間隔は変更できることが望ましい．</p>        
```

Comment:
Since https://github.com/w3c/csswg-drafts/issues/5086 hasn't been resolved yet, maybe we shouldn't insert line feeds in the Japanese version?

Reply:
ah, could be...
(actually, this was a trick in old age of Japanese web pages, to break at punctuation marks..)